### PR TITLE
463993 LA disposal cost by material

### PR DESCRIPTION
**Pull request by johnbenjamin-defra**

### CODE REVIEW

File: src/EPR.Calculator.API.UnitTests/Builder/CalcResultBuilderTests.cs [hunk 1]
```diff
@@ -18,41 +20,41 @@ public class CalcResultBuilderTests
...
-        {
-            // Act
-            var instance = new CalcResultBuilder(
-                calcResultDetailBuilder.Object,
-                lapcapBuilder.Object,
-                mockICalcResultParameterOtherCostBuilder.Object,
-                mockOnePlusFourApportionmentBuilder.Object,
-                commsCostReportBuilder.Object,
-                mockLateReportingBuilder.Object,
-                mockRunLaDisposalCostBuilder.Object);
```
Comment (udaydefra) [inline src/EPR.Calculator.API.UnitTests/Builder/CalcResultBuilderTests.cs:52]: Why is this commented?

File: src/EPR.Calculator.API.UnitTests/CalResultsTests.cs [hunk 2]
```diff
@@ -72,39 +74,40 @@ public void Setup()
...
 
         }
 
-        [TestMethod]
-        public void PrepareCalcResults_ShouldReturnCreatedStatus()
-        {
-            var requestDto = new CalcResultsRequestDto();
-            var calcResult = new CalcResult();
-            mockCalcResultBuilder.Setup(b => b.Build(requestDto)).Returns(calcResult);
+        //[TestMethod]
```
Comment (udaydefra) [inline src/EPR.Calculator.API.UnitTests/CalResultsTests.cs:85]: Any reason why this is commented?

File: src/EPR.Calculator.API.UnitTests/Builder/CalcResultBuilderTests.cs (hunk 1 above)
Comment (johnbenjamin-defra) [reply src/EPR.Calculator.API.UnitTests/Builder/CalcResultBuilderTests.cs:52]: Good catch @udaydefra, thank you. I have uncommented the tests.

File: src/EPR.Calculator.API.UnitTests/CalResultsTests.cs (hunk 2 above)
Comment (johnbenjamin-defra) [reply src/EPR.Calculator.API.UnitTests/CalResultsTests.cs]: I have uncommented the test.

**Review by udaydefra — APPROVED** [review]

**Pull request merged**